pyfar/utils.py

```diff
@@ -127,3 +127,92 @@ def broadcast_cdims(signals, cdim=None):
     if cdim is None:
         cdim = np.max([len(s.cshape) for s in signals])
     return [broadcast_cdim(s, cdim) for s in signals]
+
+
+def concatenate(signals, caxis=0, broadcasting=False, comment=""):
+    """
+    Merge multiple Signal objects along a given axis. The signals are copied,
+    if needed broadcasted in largest dimenson and to a common cshape and a new
+    object is returned.
+
+    The :py:mod:`cshape <pyfar._concepts.audio_classes>` of the signals are
+    broadcasted following the `numpy broadcasting rules
+    <https://numpy.org/doc/stable/user/basics.broadcasting.html>`_
+
+    Parameters
+    ----------
+    signals : tuple of Signal
+        The signals to concatenate. Must either have the same cshape or be
+        broadcastable to the same cshape, except in the dimension corresponding
+        to axis (the first, by default). If this is the case, set
+        ``broadcasting=True``.
+    caxis : int
+        The axis along which the signals are concatenated.
+        The default is ``0``.
+    broadcasting: bool
+        If this is ``True``, the signals will be broadcasted into largest
+        dimension and into a common cshape, except of the caxis along which the
+        signals are concatenated. The default is ``False``.
+    comment: string
+        A comment related to the merged `data`. The default is ``""``, which
+        initializes an empty string. Comments of the input signals will also
+        be returned in the new Signal object. These comments are marked with
+        their corresponding signal position in the input tuple.
```

Comment:
I would suggest to simply the comments handling. Use an empty string if the provided comment. Discard comments from input data. What do the others think?